lib/improver/tests/acceptance/test_apply_lapse_rate.py

```diff
@@ -48,7 +48,7 @@ def test_basic(tmp_path):
             kgo_dir / "ukvx_lapse_rate.nc",
             kgo_dir / "ukvx_orography.nc",
             kgo_dir / "../highres_orog.nc",
-            output_path]
+            "--output", f"{output_path}"]
```

Comment:
Do you need the `f"{output_path}"`?  I've just used `output_path` and it worked fine - does that not work in this case?

Reply:
```suggestion
            "--output", output_path]
```